CHANGELOG.rst

```diff
@@ -5,7 +5,7 @@ Changelog
 
 0.15.0
 ------
-**Date:** TBA
+**Date:** 2023-03-XY
 
 [Title]
```

Comment:
Highlights:
- ADIOS2:
  - BP5 support, ``APPEND`` and ``READ_LINEAR`` modes
  - memory footprint optimizations
- consistently configure openPMD via JSON, use TOML alternatively
- error-recovery in reads & openPMD-specific runtime exceptions
- new raw and smart-pointer API interfaces, including memory savings for ADIOS BP5
- C++17; Python 3.10 & 3.11